packages/twenty-server/src/engine/core-modules/billing/stripe/stripe.service.ts

```diff
@@ -105,4 +106,23 @@ export class StripeService {
     }
     await this.stripe.invoices.pay(latestInvoice.id);
   }
+
+  async updateBillingSubscriptionItem(
+    stripeSubscriptionItem: BillingSubscriptionItem,
+    stripePriceId: string,
+  ) {
+    try {
+      await this.stripe.subscriptionItems.update(
+        stripeSubscriptionItem.stripeSubscriptionItemId,
+        {
+          price: stripePriceId,
+          quantity: stripeSubscriptionItem.quantity,
+        },
+      );
+
+      return true;
+    } catch (e) {
+      return false;
```

Comment:
it looks suspicious to return true or false depending on the success of the update. I would let the exception or try catch but trigger a more comprehensive exception in the catch. Then we can catch it outside of the service or let it back to the graphql response

Reply:
yes you are right. Updated